modules/serverless-configuring-response-start-timeout.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/config-applications/configuring-revision-timeouts.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-configuring-resoponse-start-timeout_{context}"]
+= Configuring revision response start timeout
+
+By setting the revision response start timeout, you can specify the maximum duration in seconds that Serving waits for a revision to start sending network traffic after a request has been routed to it. The revision response start timeout must not exceed the revision timeout. The default is 300 seconds (5 minutes).
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have cluster administrator permissions on {ocp-product-title}. Alternatively, you have cluster administrator or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+
+.Procedure
+
+* Choose the appropriate method to configure the revision response start timeout:
+** To configure the timeout globally, set the `revision-response-start-timeout-seconds` field in your `KnativeServing` custom resource (CR). If your required response start timeout exceeds the revision timeout, also adjust the `revision-timeout-seconds` field accordingly:
++
+.Example of revision response start timeout globally set to 300 seconds (5 minutes)
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    defaults:
+      revision-timeout-seconds: "600"
+      revision-response-start-timeout-seconds: "300"
+----
++
+** To configure the timeout per revision, set the `responseStartTimeoutSeconds` field in your service definition. If your required response start timeout exceeds the revision timeout, also adjust the `timeoutSeconds` field accordingly:
++
+.Example of a service definition with revision response start timeout set to 300 seconds (5 minutes)
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  namespace: my-ns
+spec:
+  template:
+    spec:
+      timeoutSeconds: 600
+      responseStartTimeoutSeconds: 300
+      containers:
+# ...
+----
+
++
+[NOTE]
+====
+To set the revision response start timeout and the revision timeout to a value over 600 seconds (10 minutes), you must increase the default {ocp-product-title} route timeout and the maximum revision timeout. 
+
+For instructions on how to configure timeouts for requests exceeding the default 600 seconds (10 minutes), see "Long Running Requests".
+====
```

Comment:
If this is important to know, consider moving it to the introduction of the task. Someone using the first option might skip over this since they are skipping the second option, but it applies to either case.